editor/src/utils/deep-equality.ts

```diff
@@ -359,7 +359,7 @@ export function combine9EqualityCalls<A, B, C, D, E, F, G, H, I, X>(
   }
 }
 
-export function combine10EqualityCalls<A, B, C, D, E, F, G, H, I, J, X>(
+export function combine11EqualityCalls<A, B, C, D, E, F, G, H, I, J, K, X>(
```

Comment:
I think it'd be better if we'd also keep `combine10EqualityCalls` alive, we might need it in the future

Reply:
fixed